packages/devops/scripts/lambda/createOrRestoreDeployment.py

```diff
@@ -205,6 +184,40 @@ def create_instance(account_ids, restore_code, stage, instance_type):
     return new_instance_object
 
 
+def create_gateway(tupaia_instance_name, tupaia_instance_id, ssl_certificate_arn):
+    # 1. Create ELB
+    prod_gateway_elb = get_gateway_elb('production')
+    new_gateway_elb = create_gateway_elb(
+        tupaia_instance_name=tupaia_instance_name,
+        config=prod_gateway_elb,
+        gateway_name='gateway-elb-' + tupaia_instance_id,
```

Comment:
This could be dropped as a param and pushed inside the function (same for target group)

Reply:
Updated